src/Components/CommentForm.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import { postComment } from '../api';
+
+const CommentForm = ({ articleId, onCommentPosted }) => {
+  const [comment, setComment] = useState('');
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [error, setError] = useState(null);
+  const [successMessage, setSuccessMessage] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    
+    if (!comment.trim()) {
```

Comment:
`!commment` would achieve the same

Reply:
I guess you are trying to remove white space, so someone can't just post a space?